ui/guest/src/components/Guest.js

```diff
@@ -697,35 +691,21 @@ export function Guest(props) {
     },
 
     onAssetDragStarted(asset) {
-
       let
         players = [],
         siblings = [],
         containers = [],
-        dropZones = [];
-
-      const receptacles = iceRegistry.getMediaReceptacles();
-      const validReceptacles = receptacles.filter((id) => {
-
-        const
-          record = iceRegistry.getReferentialEntries(id),
-          validations = record.field.validations;
-
-        if (isNullOrUndefined(validations)) {
-          return false;
-        } else if (notNullOrUndefined(validations.mimeTypes)) {
-          const values = validations.mimeTypes.value;
-          return (
-            (
-              values.includes('image/*') &&
-              asset.type.includes('image/')
-            ) || (
-              values.includes(asset.type)
-            )
-          );
-        }
+        dropZones = [],
+        type;
 
-      });
+      if (asset.mimeType.includes('image/')) {
+        type = 'image';
+      } else if (asset.mimeType.includes('video/')) {
+        type = 'video-picker';
```

Comment:
what's a video-picker type?

Reply:
is the assets video type, I thought it was 'video' but it is named 'video-picker'